plugins/woocommerce/client/blocks/tests/e2e/tests/product-collection/product-collection-errors.block_theme.spec.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { Page } from '@playwright/test';
+import { test, expect, Editor, wpCLI } from '@woocommerce/e2e-utils';
+
+test.describe( 'Product Page: error notices when adding out-of-stock products', () => {
+	test( 'displays error notice when attempting to add product beyond stock limit', async ( {
+		admin,
+		editor,
+		page,
+	} ) => {
```

Comment:
Then you must add the `pageObject` here:

```suggestion
const test = base.extend< { pageObject: ProductCollectionPage } >( {
	pageObject: async ( { page, admin, editor }, use ) => {
		const pageObject = new ProductCollectionPage( {
			page,
			admin,
			editor,
		} );
		await use( pageObject );
	},
} );

test.describe( 'Product Page: error notices when adding out-of-stock products', () => {
	test( 'displays error notice when attempting to add product beyond stock limit', async ( {
		page,
		pageObject,
	} ) => {
```

You will also need to add the import at the top of the file:

```TS

/**
 * Internal dependencies
 */
import ProductCollectionPage from './product-collection.page';
```